algrtm/src/components/VideoComponent.js

```diff
@@ -0,0 +1,24 @@
+"use client";
+import { useRef } from "react";
+
+const VideoComponent = () => {
+  const iframeRef = useRef();
+  const videoUrl = "https://www.youtube.com/embed/o5BR_GvKwDg";
+
+  return (
+    <div>
+      <iframe
+        ref={iframeRef}
+        width="560"
+        height="315"
```

Comment:
The iframe should have `width='829'` and `height='469'`. In Figma, this element also has a `border-radius: 8px`